docs/en_US/nnicli_ref.md

```diff
@@ -0,0 +1,41 @@
+# NNI Client
+
+NNI client is a python API of `nnictl`, which implements the most commonly used commands. Users can use this API to control their experiments, collect experiment results and conduct advanced analyses based on experiment results in python code directly instead of using command line. Here is an example:
+
+```
+from nnicli import NNIExperiment
```

Comment:
I prefer `Experiment` instead of `NNIExperiment`

Reply:
Sounds good. Will change

Reply:
Changed.